utils/fstack.c

```diff
@@ -174,6 +177,8 @@ void setup_task_filter(char *tid_filter, struct ftrace_file_handle *handle)
 		int tid = handle->info.tids[i];
 		struct ftrace_task_handle *task = &handle->tasks[i];
 
+		prepare_task_handle(handle, task, tid);
+
```

Comment:
Here, I call `prepare_task_handle()`

Reply:
Ah, I was confused due to the recent code change.  The setup_task_handle() is not called from outside anymore.  So yes, as you said, we should make it static.  Sorry for the noise.